closure/typedefs/Manifest.js

```diff
@@ -34,7 +34,7 @@ ManifestNode.prototype.raw;
 ManifestNode.prototype.value;
 
 /** @type {string} */
-ManifestNode.prototype.debugString;
+ManifestNode.prototype.extendedInfo;
```

Comment:
this would still be `debugString`, since this is the result of manifest parsing, not the audit